### PR TITLE
Ansible tutorial TODOs from Barcelona

### PR DESCRIPTION
**Pull request by mvdbeek**
xref https://github.com/galaxyproject/training-material/issues/1809

### CODE REVIEW

File: topics/admin/tutorials/ansible/slides.html [hunk 1]
```diff
@@ -154,7 +156,7 @@
 
 ???
 
-**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server.
+**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server. Templates have a .j2 suffix, which indicates that they are Jinja 2 templates. After filling the template with the variable values, we copy the file to its remote destination without the .j2 suffix.
```
Comment (nsoranzo) [inline topics/admin/tutorials/ansible/slides.html]: I'd probably put this explanation in `### {% icon details %} Template Syntax` in the tutorial, where there is an example.

File: topics/admin/tutorials/ansible/tutorial.md [hunk 2]
```diff
@@ -307,7 +307,7 @@ The above introduction was certainly not enough for you to feel confident in Ans
 >    >
 >    {: .details }
 >
-> 8. Create a `roles/my-role/files` folder, and within it a file named `test.txt`, containing the content "Hello, World"
+> 8. Create a `roles/my-role/files` folder, and within it a file named `test.txt`, containing the content "Hello, Galaxy"
```
Comment (hexylena) [inline topics/admin/tutorials/ansible/tutorial.md]:
```suggestion
> 8. Create a `roles/my-role/files` folder, and within it a file named `test.txt`, containing the content "Hello, Galaxy 🚀"
```

File: topics/admin/tutorials/ansible/slides.html (hunk 1 above)
Comment (hexylena) [inline topics/admin/tutorials/ansible/slides.html]:
```suggestion
> ### {% icon details %} Template Syntax
> **Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server. Templates have a .j2 suffix, which indicates that they are Jinja 2 templates. After filling the template with the variable values, we copy the file to its remote destination without the .j2 suffix.
{: .details }
```
```suggestion
**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server. Templates have a .j2 suffix, which indicates that they are Jinja 2 templates. After filling the template with the variable values, we copy the file to its remote destination without the .j2 suffix.
```

Comment (mvdbeek) [reply topics/admin/tutorials/ansible/slides.html]: Yep, did that.

File: topics/admin/tutorials/ansible/slides.html [hunk 3]
```diff
@@ -154,9 +156,7 @@
 
 ???
 
-**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server.
-
-**variables** are defined for use in the templates and tasks, and **handlers** are used to automatically restart services when configuration changes.
+**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server. **variables** are defined for use in the templates and tasks, and **handlers** are used to automatically restart services when configuration changes.
```
Comment (nsoranzo) [inline topics/admin/tutorials/ansible/slides.html]:
```suggestion
**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server.

**variables** are defined for use in the templates and tasks, and **handlers** are used to automatically restart services when configuration changes.
```

**Pull request merged**